packages/cli/src/commands/doctor/healthchecks/reactNativeGlobal.ts

```diff
@@ -26,42 +30,48 @@ const findGlobalPackage = (packageManager, moduleId) => {
   }
 };
 
-const checkGlobalPaths = moduleId => {
-  return !!(
-    (findGlobalPackage(packageManagers.yarn, moduleId) ||
-      findGlobalPackage(packageManagers.npm, moduleId)) &&
-    true
-  );
+const checkGlobalInstallalations = () => {
+  //Returns an array of globally installed paths for rn and rncli both in yarn and npm
+  const librariesGloballyInstalled = Object.keys(packages)
+    .map(packageKey =>
+      Object.keys(packageManagers).map(packageManagerKey =>
+        findGlobalPackage(
+          packageManagers[packageManagerKey],
+          packages[packageKey],
+        ),
+      ),
+    )
+    .filter(e => e != null);
```

Comment:
You can replace this with:

```suggestion
    .filter(Boolean);
```